dev/change-scala-version.sh

```diff
@@ -19,7 +19,7 @@
 
 set -e
 
-VALID_VERSIONS=( 2.12 )
+VALID_VERSIONS=( 2.12 2.13 )
```

Comment:
I forgot to add this earlier; it's necessary to be able to try the 2.13 build if one wanted to locally.